app/models/theme_settings_migration.rb

```diff
@@ -4,7 +4,7 @@ class ThemeSettingsMigration < ActiveRecord::Base
   belongs_to :theme
   belongs_to :theme_field
 
-  validates :theme_id, presence: true
+  validates :theme_id, presence: true, uniqueness: { scope: :version }
```

Comment:
We need this so that ActiveRecord raises an error instead of a `PG::Error` which is much harder to handle.